pkg/aws/policies.go

```diff
@@ -123,6 +123,9 @@ var AccountRoles map[string]AccountRole = map[string]AccountRole{
 	SupportAccountRole:      {Name: "Support", Flag: "support-role-arn"},
 }
 
+var OCMUserRole = "OCM-User"
```

Comment:
Testing this and I think we should remove the `OCM-` prefix and just leave `User`. Especially if we use the default `ManagedOpenShift` prefix. It looks better this way: `arn:aws:iam::765374464689:role/ManagedOpenShift-User-vkareh-Role` and is consistent with other role prefixes (ManagedOpenShift-{role_type}-etc...)
```suggestion
var OCMUserRole = "User"
```